contact.md

```diff
@@ -9,8 +9,10 @@ header:
   actions:
     - label: "Email"
       url: "mailto:info@allhandsactive.org"
-    - label: "Visit in person"
```

Comment:
No problem with this change, but I would have liked to see it kept separate from this.